clients/benchmarks/client.cpp

```diff
@@ -522,6 +522,10 @@ try
          value<bool>(&arg.print_kernel_info)->default_value(false),
          "Print solution, kernel name and solution index.")
 
+        ("rotating",
+         value<int32_t>(&arg.rotating)->default_value(0),
+         "Use rotating memory blocks for each iteration, size in MB. (Only supports GEMM + api_method c)")
+
```

Comment:
return  error if user enable rotating buffer for non-supported cases.

Reply:
It'll just ignore instead. Change to return error?

Reply:
better to returing error to let user know, since user may expect some perf different comparing with no rotating buffer.

Reply:
Fixed